### PR TITLE
[chore] add changelogs for each component

### PR DESCRIPTION
**Pull request by krisgardiner**
When any changes are made to a component, describe the change with PR reference under the appropriate category (Breaking changes indicates major version upgrade, new features indicates minor version upgrade, and bugfixes indicates patch version upgrade).

When a release is made, it should be paired with moving the changes from `# Pending Changes` to the version it will be upgraded to. Example below:

**Before release:**
```
# Pending Changes

## Breaking

## New Features
- new feature #123

## Bug Fixes

```

**After release:**
```
# Pending Changes

## Breaking

## New Features

## Bug Fixes

# v1.1.0

## Breaking

## New Features
- new feature #123

## Bug Fixes

```

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/datepicker/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Pending Changes
+
+## Breaking
+
+## New Features
+
+## Bug Fixes
```
Comment (philrenaud) [inline components/datepicker/CHANGELOG.md:7]: Do we intend for datepicker to be a public-facing component? We don't have supporting docs for it, and have been using it internally / developing it as such. I would argue we don't want to use a changelog here.

File: components/contacts-search/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Pending Changes
+
+## Breaking
+
+## New Features
+
+## Bug Fixes
```
Comment (philrenaud) [inline components/contacts-search/CHANGELOG.md:7]: Do we intend for contacts-search to be a public-facing component? We don't have supporting docs for it, and have been using it internally / developing it as such. I would argue we don't want to use a changelog here.

**Review by philrenaud — APPROVED** [review]
LGTM. Do we populate `Pending Changes` on PRs prior to releases / on feature branches?

**Comment by krisgardiner** [comment]
@philrenaud Ideally, pending changes should be populated on the feature branch PRs so that they are always up to date.

**Pull request merged**